src/layout/section/heroSection/index.tsx

```diff
@@ -1,123 +1,168 @@
+    padding-bottom: ${space.xl};
+    margin-bottom: ${space.xl};
+  }
+`
+
+const DEPRECATED_SMALL_MEDIA_STYLES = css`
+  & {
+    margin-bottom: 0;
+    height: initial;
+  }
+
+  & .hero-image {
+    position: static;
+    height: 136px;
+    background-image: ${(props: HeroSectionProps) => `url(${props.heroImageUrl})`};
+  }
+
+  & .hero-image::after {
+    content: none;
+  }
+
+  & .hero-info {
+    padding-top: ${space.xl};
+  }
+
+  & .hero-info-title {
+    color: ${color.primaryText};
+  }
+
+  & .hero-info-text {
+    color: ${color.secondaryText};
+    font-size: 16px;
+  }
+
+  & .hero-content {
+    align-items: center;
+  }
+`
 
 const StyledHeroSection = styled(HeroSection)`
   & {
     background-color: ${color.defaultBackground};
     position: relative;
     z-index: 1;
-    padding: 0;
+    height: 75vh;
+    margin-bottom: ${space.xl};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
   }
 
   & .hero-image {
-    position: relative;
-    min-height: 136px;
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
     background-size: cover;
     background-position: center;
-    background-image: ${props => `url(${props.heroImageUrl})`};
+    background-image: ${props => `url(${props.heroImageUrlLarge})`};
+
+    /* Hide the shadow. */
+    overflow: hidden;
+  }
+
+  & .hero-image::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 75%;
+    display: block;
+    background: linear-gradient(
+      0deg,
+      rgba(0, 0, 0, 0.4) 72%,
+      rgba(0, 0, 0, 0.2) 85%,
+      rgba(0, 0, 0, 0) 100%
+    );
   }
 
   & .hero-info {
-    text-align: center;
-    margin-left: auto;
-    margin-right: auto;
+    width: 100%;
     max-width: ${componentSizes.largeSectionWidth};
-    padding: ${space.xl};
-    padding-top: 0;
+    padding: 0 ${space.xl};
+    padding-top: 120px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
   }
 
   & .hero-info-title-wrapper {
-    display: flex;
-    flex-direction: column;
+    text-align: center;
   }
 
-  & .hero-info-title.kirk-title {
-    min-height: auto;
-    align-items: flex-end;
-    align-self: center;
-    display: flex;
-    padding-top: ${space.xl};
-    padding-bottom: ${space.ms};
+  & .hero-info-title {
     margin: 0;
+    padding-top: ${space.m};
+    padding-bottom: ${space.m};
+    color: ${color.textWithBackground};
+    font-size: 60px;
+  }
+
+  & .hero-info-text {
+    font-size: 22px;
+    color: ${color.textWithBackground};
   }
 
-  & .hero-button {
+  & .hero-content {
+    width: 100%;
     margin-top: ${space.xl};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
   }
 
-  @media (${responsiveBreakpoints.isMediaLarge}) {
+  ${props => (props.searchForm == null ? DEPRECATED_STYLES : null)}
+
+  @media (${responsiveBreakpoints.isMediaSmall}) {
```

Comment:
would be nicier as mobile first

Reply:
I can do that but there will be a bit more style overrides for the deprecated part.

Reply:
:(

Reply:
I'll still do it as it only affect the deprecated part.